atlp-rwanda/commanders-rn-medica
# Feature: doctors screen

**What does this PR do?**

This PR includes designs of favorite doctors, top doctors and list of all doctors where you can filter and find any doctor registered in the system of the app

**Description of Task to be completed?**

Add photos of the doctors and their information including reveiws and the hospitals their work on and their roles
create needed components for the designs like the cards ,buttons


**How should this be manually tested?**

You will click on the **heart icon** on the Home screen to go to **favorite screen** and on the favorite doctor screen then click on one of the cards where you will go to a screen of removing a doctor from the favoritates .On the home screen too you will click on the **see all** which is next to top doctors and you will be redirected to the **top doctor screen** ,there you will click on the search icon where you will be directed to the search screen which will load and after sometime you will be redirected to the not found screen and  on the not found you can click on  **deafult** you will be directed to a page where doctors were found and their  list and when you click on **default**  you will be directed to the filter screen .

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker/Trello stories?**

https://github.com/atlp-rwanda/commanders-rn-medica/issues/19

**Screenshoots of the screens designed in this pr**

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/ee6a9eef-0ab5-4e52-8106-6aa7c86f08fc">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/44896712-1bc6-437b-86fc-5797cbe01f59 ">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/132d9fd4-fc12-43fb-9323-1048893522b9">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/ba0da711-e5dc-4919-93e7-4dcac802ebee">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/31b60dce-5d95-4764-9ca5-f2da814f15b2">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/2f71f81d-f236-45a8-b9e3-0f908c1fe4cd">

<img width="150" alt="Screenshot 2024-05-10 at 14 00 08" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/99607224/5da71b76-74e0-4505-959e-58e388f644fd">
